release.json

```diff
@@ -8,7 +8,7 @@
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
         "OMNIBUS_SOFTWARE_VERSION": "8664357efd9187e1c16284e3d9811ea25d152186",
-        "OMNIBUS_RUBY_VERSION": "78573b961ef3400787a7069526d91795c1a34238",
+        "OMNIBUS_RUBY_VERSION": "chouquette/tmp_split_build_packaging",
```

Comment:
Is this the desired change or something temporary?

Reply:
Temporary, until the required changes in omnibus ruby are merged